less/mixins.less

```diff
@@ -1 +1,16 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+
+.font-nav-and-h1{
+    font-family: 'Kenia', cursive;
+}
+
+.button{
+    
+     width: 100%;
+     height: 10vh;
```

Comment:
Great idea for use of a mixin here. You can really leverage the power of the mixin too if you wanted to add your animation effects, and even break points nested in the mixins you can!